mungegithub/github/github.go

```diff
@@ -1289,6 +1292,41 @@ func (obj *MungeObject) MergePR(who string) error {
 }
 
 // ListComments returns all comments for the issue/PR in question
+func (obj *MungeObject) ListPRComments() ([]github.PullRequestComment, error) {
```

Comment:
OK, I'm going to change my names a little bit. There are two types of comments:
- Issue comments
- Review comments: these are related to a specific line of code in the diff